images/macos/provision/core/toolset.ps1

```diff
@@ -38,7 +38,7 @@ $tools =  Get-ToolcacheFromToolset | Where-Object {$ToolsToInstall -contains $_.
 
 foreach ($tool in $tools) {
     # Get versions manifest for current tool
-    $assets = Invoke-RestMethod $tool.url
+    $assets = Invoke-RestMethod $tool.url -MaximumRetryCount 10 -RetryIntervalSec 30

```

Comment:
Do we want to add some helper function as a wrapper of `Invoke-RestMethod $tool.url -MaximumRetryCount 10 -RetryIntervalSec 30`. Just to make sure that we have these hardcoded values in one place instead of every script?

Reply:
Sounds good, but we have only `toolset.ps1` as pwsh script on macOS. Does it make sense?

Reply:
Done

Reply:
I guess we will have more in future :)